packages/devextreme/scss/widgets/fluent/scheduler/_colors.scss

```diff
@@ -109,6 +112,11 @@ $header-panel-cell-date: null !default;
   $scheduler-appointment-text-color: #fff !default;
   $scheduler-panel-text-color: #999;
   $header-panel-cell-date: #adadad;
+
+  @if $color == "saas" {
+    $scheduler-workspace-focused-cell-color: darken(desaturate(adjust-hue($base-accent, -7), 65.00), 41.18);
```

Comment:
If we need override variable value, we should declare variable
```$scheduler-workspace-focused-cell-color: null !default; ``` 
and after that for each mode and color set required value.